WcaOnRails/app/models/competition.rb

```diff
@@ -640,7 +640,11 @@ def in_progress?
   end
 
   def is_over?
-    !end_date.nil? && end_date < Date.today
+    # We treat "today == last day of competition" as the competition is over.
+    # Once we know the schedules and timezones of competitions, we'll be able
+    # to do something more accurate here.
+    # See https://github.com/thewca/worldcubeassociation.org/issues/737
+    !end_date.nil? && end_date <= Date.today
```

Comment:
Thanks! That explains a lot.

What I didn't understand is that this is inherently inaccurate, so we're making a rough guess. The comment could probably be clearer on that. Or renamed, but I can't think of a good name. `competition.is_probably_over?` doesn't feel like a bullseye.

But do we really have to do this at all? Don't we trust delegates to follow and enforce far more serious rules? Why police this small thing, especially since it makes their job harder and complicates out code?